docs/source/formulations_example.rst

```diff
@@ -302,7 +302,7 @@ The code below defines the rows and defines one column that contains the identif
         datasets=[dataset.uid],
         rows=[
             MaterialRunByTemplate(
-                templates=[LinkByUID.from_entity(t) for t in material_templates_to_include]
+                templates=[LinkByUID.from_entity(t, scope) for t in material_templates_to_include]
```

Comment:
This is a correct fix, but it raises the question of does it make sense for citrine-python to replace the default scope on import.  That might be more dark-art/monkey-patch than you feel comfortable with.

Reply:
Not sure I follow. The templates are defined with this scope, so nothing is being modified